routes/golfer_solution.go

```diff
@@ -50,7 +50,7 @@ func golferSolutionGET(w http.ResponseWriter, r *http.Request) {
 }
 
 // POST /golfers/{golfer}/{hole}/{lang}/{scoring}
-//nolint deadcode
+// nolint deadcode
```

Comment:
Oh interesting, I thought it had to have no space to work, like the `//go:` ones, weird.